readme-vars.yml

```diff
@@ -0,0 +1,50 @@
+---
+
+# project information
+project_name: grav
+project_url: "https://github.com/getgrav/grav/"
+project_logo: "https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/grav-logo.png"
+project_blurb: |
+  [{{ project_name|capitalize }}]({{ project_url }}) is a Fast, Simple, and Flexible, file-based Web-platform.
+
+project_lsio_github_repo_url: "https://github.com/linuxserver/docker-{{ project_name }}"
+
+# supported architectures
+available_architectures:
+  - { arch: "{{ arch_x86_64 }}", tag: "amd64-latest"}
+  - { arch: "{{ arch_arm64 }}", tag: "arm64v8-latest"}
+  - { arch: "{{ arch_armhf }}", tag: "arm32v7-latest"}
+
+# development version
+development_versions: true
+development_versions_items:
+  - { tag: "latest", desc: "Stable releases from GitHub" }
```

Comment:
Are we planning to release development versions?

Reply:
Probably not as they don't do pre-releases as far as I can tell. Is that just a change to
```
# development version
development_versions: false
```
?

Reply:
You can remove all of these lines actually.

Reply:
Is that the same with
``
# optional parameters
opt_param_usage_include_env: false
opt_param_usage_include_vols: false
```
?

Reply:
Technically, yes. they don't cause harm if they exist and are set to false, but they aren't required.